stable/signalfx-agent/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: The SignalFx Kubernetes agent
 name: signalfx-agent
-appVersion: 3.2.2
-version: 0.2.1
+appVersion: 3.6.1
+version: 0.2.2
```

Comment:
humm, since you are adding a new requirement in the clusterrole, I would say to bump the chart version to 0.3.0 since this is not a bug fix

Reply:
Ok, yes I agree, fixed.